.prettierignore

```diff
@@ -7,3 +7,6 @@ components/src/components/icons/generated/
 
 .yarn/*
 yarn.lock
+
+icons
+*.svg
```

Comment:
why are these being added to the prettierignore?

Reply:
I was getting an error in the commit pre-hook from prettier. Something like `no parser found for svg files`

Reply:
might just wanna rm this for now and decrease the scope since the rest of the icon stuff was reverted